restful_api_with_data_mapper/main.bal

```diff
@@ -0,0 +1,95 @@
+import ballerina/http;
+
+type Person record {
+    readonly string id;
+    string firstName;
+    string lastName;
+    int age;
+    string country;
+};
+
+type Course record {
+    string id;
+    string name;
+    int credits;
+};
+
+type Student record {
+    string id;
+    string fullName;
+    string age;
+    record {
+        string title;
+        int credits;
+    }[] courses;
+    int totalCredits;
+    string visaType;
+};
+
+const D_TIER_4_VISA = "D tier-4";
+
+var totalCredits = function(int total, record {string id; string name; int credits;} course) returns int => total + (course.id.startsWith("CS") ? course.credits : 0);
```

Comment:
Break the line, usually 120 char limit.

Reply:
Fixed via [74db4ed](https://github.com/ballerina-guides/integration-samples/pull/27/commits/74db4ed66a889c18321790d55a4092e0647cf9b3)